app/frontend/actions.ts

```diff
@@ -1014,27 +1019,31 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
       setState({
         shelleyDelegation: {
           ...state.shelleyDelegation,
-          delegationFee: 0,
+          delegationFee: 0 as Lovelace,
         },
       })
     }
   }
 
-  const updateStakePoolIdentifier = (state, e, hash?) => {
+  const updateStakePoolIdentifier = (
+    state: State,
+    poolHash: string,
+    validationError: any = null
+  ): void => {
     /**
      * this has to be redone,
      * pool validation must happen before debouncing
      * but pool info shown after
      */
-    const poolHash = hash || e.target.value
-    const validationError = poolIdValidator(poolHash, state.validStakepools)
+    const newPool = state.validStakepools.getPoolInfoByPoolHash(poolHash)
+    const oldPool = state.shelleyDelegation.selectedPool
+    if (newPool && oldPool && newPool.poolHash === oldPool.poolHash) return
```

Comment:
We can use optional chaining here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
to be clear it's not 100% equivalent - but if both newPool and oldPool are null, setting the newPool doesn't really make sense

Reply:
Optional chaining doesn't work with our setup of prettier. Don't know how to make it work. For now I just ignored prettier errors.

Reply:
@refi93 I tried to get this working as well - I updated all dependencies, re-installed node modules, etc., without success. I updated the node too. Do you know how to get this to work?

Reply:
we can look into it, I see `yarn fix` is failing with it, this issue might be related: https://github.com/prettier/prettier/issues/6609